DataCenterBridging.psd1

```diff
@@ -12,7 +12,7 @@
 RootModule = 'DataCenterBridging.psm1'
 
 # Version number of this module.
-ModuleVersion = '2021.6.23.36'
+ModuleVersion = '2021.7.14'

```

Comment:
This is date based and updated by the build. No need to update as this will be a no-op

Reply:
Thanks for letting me know. I'll leave the psd1 alone in the future.